vscode-react/src/sections/aKDeployments.section.tsx

```diff
@@ -145,6 +217,54 @@ export const AKDeployments = ({
 										onClick={() => deactivateBuild(deployment.deploymentId)}
 									></div>
 								)}
+								<div
+									className="codicon codicon-trash cursor-pointer ml-2"
+									onClick={(e) => toggleDeletePopper(e.currentTarget, deployment.deploymentId)}
```

Comment:
Instead of toggle, close the current and open the relevant popper for the new click.